### ISSUE TITLE
Internal application error in join form

### ISSUE THREAD

**Issue by pbenety** [zerocracy/farm#891]
I've got an error while submitting the join form.

## Expected Behavior
Inform that the field "StackOverflow ID" is required.

## Current Behavior
Internal application error.

## Steps to Reproduce
1. Fill all fields but "StackOverflow ID" field.
2. Submit form.

## Stack trace
```
java.lang.IllegalStateException: [POST https://www.0crat.com/join-post] failed in 53ms: For input string: ""
	at org.takes.facets.fallback.TkFallback.error(TkFallback.java:234)
	at org.takes.facets.fallback.TkFallback.route(TkFallback.java:115)
	at org.takes.facets.fallback.TkFallback.access$000(TkFallback.java:54)
	at org.takes.facets.fallback.TkFallback$1.act(TkFallback.java:66)
	at org.takes.tk.TkWrap.act(TkWrap.java:61)
	at org.takes.tk.TkSslOnly.act(TkSslOnly.java:71)
	at org.takes.tk.TkWrap.act(TkWrap.java:61)
	at org.takes.http.BkBasic.print(BkBasic.java:108)
	at org.takes.http.BkBasic.accept(BkBasic.java:84)
	at org.takes.http.BkSafe$1.accept(BkSafe.java:51)
	at org.takes.http.BkWrap.accept(BkWrap.java:52)
	at org.takes.http.BkTimeable.accept(BkTimeable.java:89)
	at org.takes.http.BkParallel$1$1.run(BkParallel.java:89)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NumberFormatException: For input string: ""
	at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
	at java.lang.Integer.parseInt(Integer.java:592)
	at java.lang.Integer.parseInt(Integer.java:615)
	at com.zerocracy.tk.TkJoin.act(TkJoin.java:74)
	at org.takes.facets.fork.FkRegex.route(FkRegex.java:210)
	at org.takes.facets.fork.FkChain.route(FkChain.java:75)
	at org.takes.facets.fork.TkFork.act(TkFork.java:100)
	at org.takes.facets.forward.TkForward.act(TkForward.java:68)
	at org.takes.facets.fork.FkFixed$1.route(FkFixed.java:56)
	at org.takes.facets.fork.FkWrap.route(FkWrap.java:61)
	at org.takes.facets.fork.FkChain.route(FkChain.java:75)
	at org.takes.facets.fork.TkFork.act(TkFork.java:100)
	at org.takes.facets.auth.TkAuth.act(TkAuth.java:109)
	at org.takes.facets.auth.TkAuth.act(TkAuth.java:89)
	at org.takes.tk.TkWrap.act(TkWrap.java:61)
	at org.takes.facets.flash.TkFlash.act(TkFlash.java:108)
	at org.takes.tk.TkGzip$1.act(TkGzip.java:58)
	at org.takes.tk.TkWrap.act(TkWrap.java:61)
	at org.takes.tk.TkMeasured$1.act(TkMeasured.java:67)
	at org.takes.tk.TkWrap.act(TkWrap.java:61)
	at org.takes.tk.TkVersioned$1.act(TkVersioned.java:73)
	at org.takes.tk.TkWrap.act(TkWrap.java:61)
	at org.takes.tk.TkWithHeaders$1.act(TkWithHeaders.java:71)
	at org.takes.tk.TkWrap.act(TkWrap.java:61)
	at org.takes.facets.fallback.TkFallback.route(TkFallback.java:86)
	... 14 more
``` 

## Version
1.0-SNAPSHOT, 8699e2ef

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by 0crat** [comment]
@pbenety[/z](https://www.0crat.com/u/pbenety) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C3NDPUA8L) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by yegor256** [comment]
@pbenety fixed in 7e1e1096 thanks!

**Comment by 0crat** [comment]
The job is not in WBS, won't close the order
